models/github_user.go

```diff
@@ -0,0 +1,14 @@
+package models
```

Comment:
Nitpicking here, the good practice in Go is to name the packages in singular.
```suggestion
package model
```

Reply:
thanks for the tip! updated it